Dockerfile.openpilot_base

```diff
@@ -25,8 +25,9 @@ RUN cd /tmp && \
     rm -rf /tmp/* && \
     rm -rf /root/.cache && \
     pip uninstall -y pipenv && \
-
     # remove unused architectures from gcc for panda
     cd /usr/lib/gcc/arm-none-eabi/9.2.1 && \
     rm -rf arm/ && \
-    rm -rf thumb/nofp thumb/v6* thumb/v8* thumb/v7+fp thumb/v7-r+fp.sp
+    rm -rf thumb/nofp thumb/v6* thumb/v8* thumb/v7+fp thumb/v7-r+fp.sp && \
+    sudo git config --global --add safe.directory /tmp/openpilot
```

Comment:
Hopefully a fix for https://github.blog/2022-04-12-git-security-vulnerability-announced/